text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -62,11 +62,13 @@ model:
     insert_sep: True
     text_segment_num: 2
     stochastic_chunk: False
-    text_detection_length: 10                          # for augmentation purpose: naive way to different text column versus tabular column that are treated as text
+    text_aug_detect_length: 10                          # We perform text augmentation only if a text has more than text_detection_length words. It is used to differentiate text columns versus tabular columns that are treated as text.
     text_train_augment_types:                          # https://arxiv.org/abs/1901.11196; 
       # example
+      - "trivial_augment({'max':0.1})"            # max is the maximum percent of word can be changed for a text
```

Comment:
Should we use the same style as AutoAugment? https://github.com/rwightman/pytorch-image-models/blob/db8e33c69f20032298e428cac477334f7f166597/timm/data/auto_augment.py#L676 

> Ex 'rand-m9-n3-mstd0.5' results in RandAugment with magnitude 9, num_layers 3, magnitude_std 0.5

Reply:
Ex 'rand-m9-n3-mstd0.5' only specify the parameter for random augment, but not what operation. The operations are predefined here https://github.com/rwightman/pytorch-image-models/blob/db8e33c69f20032298e428cac477334f7f166597/timm/data/auto_augment.py#L563

Reply:
Do you think if it makes sense for us to adopt a similar logic?

Reply:
We are using the same style as AutoAugment. AutoAugment doesn't allow user to specify all transformation.  And, the only parameter trivial augment take is the maximum scale, and user specify as "trivial_augment({'max':0.1})" , which is similar to "rand-m9-n3-mstd0.5.

Reply:
I just feel that the string is too long and it involves quotation mark.

Reply:
Or `trivial-max0.9`, `trivial`.

Reply:
What do you think if we use `trivial_augment(0.9)`

Reply:
I think we can consider to support `trivial_augment(0.9)` also. When we are parsing it with `literal_eval`, it will be 0.9 and can be treated as `args`.